sdks/js/src/index.ts

```diff
@@ -709,6 +711,34 @@ export class DustAPI {
     return new Ok({ eventStream: streamEvents() });
   }
 
+  async cancelMessageGeneration({
+    conversationId,
+    messageIds,
+  }: {
+    conversationId: string;
+    messageIds: string[];
+  }) {
+    const res = await this.request({
+      method: "POST",
+      path: `assistant/conversations/${conversationId}/cancel`,
+      body: {
+        action: "cancel",
```

Comment:
I know it's coming from the private endpoint, but is it useful to have an action here if we already have the action name in the url ?

Reply:
Fair, let me look at the existing conversation endpoint.

Reply:
I removed the "action" in the payload :)